invenio.cfg

```diff
@@ -283,3 +283,6 @@ RDM_CUSTOM_FIELDS = [
     # obo
     RelationshipListCF(name='obo:RO_0002453'),
 ]
+
+APP_RDM_DEPOSIT_REQUIRE_FILES = True
+APP_RDM_DEPOSIT_FULL_RECORD_VISIBILITY = False
```

Comment:
is there any reason why we add config variables instead of using overridable ?

Reply:
After conversation with @slint : we agreed to use Overridable.parametrize function to overwrite the components' props instead of adding a config variable, since config variables should be reserved for full features, not only disabling the frontend components. The solution proposed in these PRs does not provide the backend validation, only hides the access field from the user (while it is still possible to bypass in REST API) there should not be any flag indicating it is a full feature, untill we provide a proper implementation, which includes records permission policy.
Actionable now: change the implementation to use overridable's parametrise function.
The overriden component code should be added in Zenodo's equivalent of https://github.com/inveniosoftware/cookiecutter-invenio-rdm/tree/master/%7B%7Bcookiecutter.project_shortname%7D%7D/site/%7B%7Bcookiecutter.package_name%7D%7D/assets/semantic-ui/js/%7B%7Bcookiecutter.package_name%7D%7D